### ISSUE TITLE
[Website] [.org+.rocks domains] Docs taking to wrong directing

### ISSUE THREAD

**Issue by FenzDev** [MonoGame/MonoGame#7521]
Hi MonoGame,
As mentioned in the title there is wrong directing to docs, To make it short here is the thing:
![image](https://user-images.githubusercontent.com/68622361/121362442-fec58180-c92d-11eb-92c5-0c01c35b6677.png)

The current link -> https://docs.monogame.org/
The actual link -> https://docs.monogame.net/

**Comment by tomspilman** [comment]
@Jjagg - I assume this is because how the `docs` subdomain is setup it doesn't appear under our other MG TLD like monogame.org or monogame.rocks.

Any way we can fix this on the server simply to always have that subdomain appear under all MG webserver domains?

**Comment by mrhelmut** [comment]
@Jjagg Couldn't we have an url rewrite rule for the .org to redirect to .net for all pages and subdomains?

**Comment by Jjagg** [comment]
@mrhelmut Yes, we can definitely do that. We host on each domain now because we didn't own the monogame.net domain and that has caused some DNS issues in the past, but Tom recently got ownership of all MG domains :) 

I'll look into it!
